consensus/dman/actors_test.go

```diff
@@ -74,9 +76,15 @@ type testingProxy struct {
 	skipCallCheck bool
 }
 
+// assert struct `testingProxy` implements `reqBusView` , `txMarshaller`, `databaseView` and `typeProxyIface` interfaces
```

Comment:
question: Does it matter if `testingProxy` doesn't implement these interfaces if tests cover that functionality?